crp_APL/interfaces/plannerInterfaces/plannerInterfaces.hpp

```diff
@@ -0,0 +1,135 @@
+#ifndef CRP_APL_PLANNER_INTERFACES_INTERFACES_HPP
+#define CRP_APL_PLANNER_INTERFACES_INTERFACES_HPP
+
+#include <vector>
+#include <cstdint>
+
+
+namespace crp
+{
+namespace apl
+{
+
+struct Vector2D
+{
+    float x{0.0f};
+    float y{0.0f};
+};
+
+struct Vector3D
+{
+    float x{0.0f};
+    float y{0.0f};
+    float z{0.0f};
+};
+
+typedef Vector2D Point2D;
```

Comment:
probably I would use Point3D everywhere - and no need of 2d separately (we set Z=0 everywhere we dont need it)

Reply:
btw, why do we use typedef here?

Reply:
Dimensions use the same 3 variables (x,y,z). I don't want to use Point3D type for dimensions (object detection: shape) or Vector3D type for points/trajectory so I created it as an alias.